sys/freebsd/sys.txt

```diff
@@ -19,6 +19,7 @@ include <sys/resource.h>
 include <time.h>
 include <signal.h>
 include <sys/wait.h>
+#include <sys/time.h>
```

Comment:
Why is there a `#` here?

Reply:
Mistake on my end. Fixed.

Reply:
Thanks. Please squash it into the previous commit, no need to have 2 commits.